pkg/api/types.go

```diff
@@ -844,6 +953,8 @@ const (
 	ResourceCPU ResourceName = "cpu"
 	// Memory, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
 	ResourceMemory ResourceName = "memory"
+	// volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 1024 * 1024)
+	ResourceStorage ResourceName = "storage"
```

Comment:
I am anxious about overly generic word here - what happens if a user submits a pod that asks for "10 GB of storage" - how will we resolve that?

@vmarmol @rjnagal